### PR TITLE
integration_tests: add first test including the controller

### PR DESCRIPTION
**Pull request by mvdan**
(see commit message)

### CODE REVIEW

**Comment by mvdan** [comment]
Also, it would help if one of you can run it locally and confirm that it works for you too.

We should also add it to CI; I didn't do that yet because each script starts its own devnet, and that's really slow. We should first refactor `header.sh` to start one devnet for all the integration tests.

File: integration_tests/header.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@
 my_dir="$(dirname "$0")"
 
 #set -o errexit ... re-enable when we fix the storage deal bug on Lotus side
-set -x
```
Comment (willscott) [inline integration_tests/header.sh:6]: don't we want this?

**Review by willscott — APPROVED** [review]

File: integration_tests/header.sh (hunk 1 above)
Comment (mvdan) [reply integration_tests/header.sh:6]: See the last part of the commit message.

**Comment by mvdan** [comment]
This is old and I have more changes now, such as a better devnet and the ability to tunnel into the new AWS machine. Closing and will open a new PR.

**Pull request closed without merging**